cfpb/consumerfinance.gov
# Add InfoUnitGroups to Expandables

This commit adds the ability to put an InfoUnitGroup module inside of an Expandable module.

This addresses internal el-camino#617.

## How to test this PR

To test, create a new page that supports Expandables (for example a BrowsePage), add an Expandable, and note that you can now add an InfoUnitGroup as its child.

## Screenshots

<img width="1002" alt="image" src="https://user-images.githubusercontent.com/654645/173105464-734e9929-4071-4f8b-ac75-94510a159484.png">
